stl/inc/xatomic_wait.h

```diff
@@ -48,8 +48,10 @@ void __stdcall __std_atomic_notify_all_direct(const void* _Storage) noexcept;
 // The "indirect" functions are used when the size is not 1, 2, 4, or 8; these notionally wait on another value which is
 // of one of those sizes whose value changes upon notify, hence "indirect". (As of 2020-07-24, this always uses the
 // fallback SRWLOCK and CONDITION_VARIABLE implementation but that is not contractual.)
-int __stdcall __std_atomic_wait_indirect(
-    const void* _Storage, const void* _Comparand, size_t _Size, unsigned long _Remaining_timeout) noexcept;
+using _Atomic_wait_indirect_callback = bool(__stdcall*)(const void* _Parameter, const void* _Comparand);

```

Comment:
This type is named identically to the function in `<atomic>`. I think we should use a different name. (Anything that resembles shadowing, whether technically shadowing or not, is confusing and interferes with searching.) I would be happy with an `_t` suffix, or anything else really.